jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/impl/model/TaskEventImpl.java

```diff
@@ -113,7 +115,22 @@ public TaskEventImpl(Long taskId, TaskEventType type, Long processInstanceId, Lo
   public TaskEventImpl(Long taskId, TaskEventType type, Long processInstanceId, Long workItemId, String userId, String message) {
     this(taskId, type, processInstanceId, workItemId, userId, new Date());
     this.message = message;
+  }
+  
+  public TaskEventImpl(org.kie.api.task.TaskEvent event, TaskEventType type) {
+      this(event, type, null);
+  }
 
+  public TaskEventImpl(org.kie.api.task.TaskEvent event, TaskEventType type, String message) {
+      InternalTaskData taskData = (InternalTaskData)event.getTask().getTaskData(); 
+      this.taskId = event.getTask().getId();
+      this.type = type;
+      this.processInstanceId = taskData.getProcessInstanceId();
+      this.workItemId = taskData.getWorkItemId();
+      this.currentOwner = taskData.getActualOwner().getId();
```

Comment:
It may generate a NPE, for example with this test implementation

https://github.com/kiegroup/jbpm/blob/6039d810fd4084fa23e4eb77c9e300499d3b1767/jbpm-persistence/jbpm-persistence-api/src/test/java/org/jbpm/persistence/api/integration/base/BaseEventCollectionTest.java#L283

```suggestion
      this.currentOwner = taskData.getActualOwner()!= null? taskData.getActualOwner().getId() : "";
```